website/docs/feature-flag-tutorials/python/implementing-feature-flags.md

```diff
@@ -37,6 +37,9 @@ In this tutorial, you will need the following:
 
 ![An architectural diagram of our Python app using Unleash feature flags](/img/python-flask-unleash-architecture.png)
 
+This architecture diagram breaks down how the Python app works with Unleash to control feature flags. We connect the Unleash service to your Python app using the Python SDK.
+
+The Unleash Server is a **Feature Flag Control Service**g, which is a service that manages your feature flags and is used to retrieve flag data from (and send data to, especially when not using a UI). The Unleash server has a UI for creating and managing projects and feature flags. There are also [API commands available](https://docs.getunleash.io/reference/api/unleash) to perform the same actions straight from your CLI or server-side app.
```

Comment:
Think a little "g" slipped in here!

```suggestion
The Unleash Server is a **Feature Flag Control Service**, which is a service that manages your feature flags and is used to retrieve flag data from (and send data to, especially when not using a UI). The Unleash server has a UI for creating and managing projects and feature flags. There are also [API commands available](https://docs.getunleash.io/reference/api/unleash) to perform the same actions straight from your CLI or server-side app.
```

Reply:
I had those sprinkled everywhere somehow along the way... loose fingers!